cpp/unreal_plugins/CoreUtils/Source/CoreUtils/CoreUtils.cpp

```diff
@@ -15,7 +15,7 @@ void CoreUtils::StartupModule()
     Config::initialize();
 
     // Wait for keyboard input, which is useful when attempting to attach a debugger to the running executable.
-    if (Config::get<bool>("CORE_UTILS.WAIT_FOR_KEYBOARD_INPUT_DURING_INITIALIZATION")) {
+    if (Config::isEnabled() && Config::get<bool>("CORE_UTILS.WAIT_FOR_KEYBOARD_INPUT_DURING_INITIALIZATION")) {
```

Comment:
Better to structure this check as an early return statement, because then it will match the other code in our all of the constructors etc.